misk-hibernate/src/main/kotlin/misk/hibernate/Transacter.kt

```diff
@@ -50,6 +50,28 @@ fun Transacter.shards() = transaction { it.shards() }
 
 fun Transacter.shards(keyspace: Keyspace) = transaction { it.shards(keyspace) }
 
+
+/**
+ * Commits a transaction with operations defined in the [lambda] block.
+ *
+ * New objects must be persisted with an explicit call to [Session.save].
+ * Updates are performed implicitly by modifying objects returned from a query.
+ *
+ * For example if we were to save a new movie to a movie database, and update the gross revenue of
+ * existing movie:
+ * ```
+ * transaction.transaction { session ->
+ *   // Saving a new entity to the database needs an explicit call.
+ *   session.save(starWars)
```

Comment:
This example needs a `new` call. Otherwise the saved instance comes from outside the transaction, which we forbid

Reply:
Fixed